rds.tf

```diff
@@ -88,7 +88,7 @@ resource "aws_security_group_rule" "allow_db_out" {
 
 resource "aws_db_instance" "postgres" {
   identifier                      = "fms-postgres-${local.naming_suffix}"
-  allocated_storage               = 60
+  allocated_storage               = 80
   storage_type                    = "gp2"
   engine                          = "postgres"
   engine_version                  = var.environment == "prod" ? "10.13" : "10.13"
```

Comment:
I believe the version will now need to be updated from 10.13 to 14.7

Reply:
It is already migrated to v14.7, this suggestion is to not cause any errors on the dq-tf-infra